docs/resources/token.md

```diff
@@ -9,7 +9,23 @@ description: |-
 
 
 
-
+## Example Usage
+
+```terraform
```

Comment:
the example should be put in a file `examples/resource/authentik_token/resource.tf` as this markdown file is auto-generated

Reply:
Thanks. Updated in the correct location.